server/chat.ts

```diff
@@ -1339,6 +1340,20 @@ export class CommandContext extends MessageContext {
 	requiresRoom() {
 		this.errorReply(`/${this.cmd} - must be used in a chat room, not a ${this.pmTarget ? "PM" : "console"}`);
 	}
+	secretCommand() {
+		const handler = this.handler;
+		handler!.isPrivate = true;
+		const full = this.fullCmd;
+		const token = this.cmdToken;
+		if (token === '!') {
+			return this.errorReply(`The command "${token}${this.cmd}" does not exist.`);
+		}
+		this.errorReply(
+			`The command "${token}${this.cmd}" does not exist. ` +
+			`To send a message starting with "${token}${full}", ` +
+			`type "${token.repeat(2)}${full}".`
```

Comment:
```suggestion
			`type "${token}${token}${full}".`
```
I'm surprised `repeat` exists now, but I don't think it actually adds to the readability here.

Reply:
Also, these should be `throw new Chat.ErrorMessage` right?